### DISCUSSION TITLE
Do not  auto-close brackets when inside double quotes or a comment

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7790] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Do-not--auto-close-brackets-when-inside-/10534365)._

---
[severity:It bothers me. A fix would be nice]
When writing Python code, and I create a Regex and a comment block and use a parenthesis or square bracket, it auto-closes it.  This should only happen in code, not inside double quotes or a comment block.

---
#### Feedback Bot on 12/5/2023, 06:51 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by StellaHuang95** [comment]
Thanks for the issue. Moving this issue to discussion as an enhancement request for comments and upvotes.
